crossdock/server/echo.go

```diff
@@ -0,0 +1,21 @@
+package server
+
+import (
+	"github.com/yarpc/yarpc-go"
+	"golang.org/x/net/context"
+)
+
+// EchoRequest contains a message to echo
+type EchoRequest struct {
+	Token string `json:"token"`
+}
+
+// EchoResponse contains a messaged echoed by EchoRequest
+type EchoResponse struct {
+	Token string `json:"token"`
+}
+
+// Echo echoes an EchoResponse
+func Echo(ctx context.Context, meta yarpc.Meta, req *EchoRequest) (*EchoResponse, yarpc.Meta, error) {
```

Comment:
Actually, we're going to do something like:

```
resp, rbody, err := Handler(ctx, req, body)
```

Having the body stand on its own reducing code generation cruft.